app/views/partials/_home_icons.html.erb

```diff
@@ -0,0 +1,67 @@
+<div class="container mt-5">
+  <div style="text-align: center">
+  <h3>You bring the brains, we’ll take care of everything else.
+</h3>
+  <h1> The greatest learning experience, join us to discover more.</h1>
+</div>
+  <div class="row mt-5">
+    <div class="col-sm-6">
```

Comment:
col-12 col-md-6
we want it to have only one column in smartphones, right?

Reply:
right, I think it's working like this right now